## tl;dr

Mixed adjustment leaves the Exchequer cost in a relatively narrow band, while poverty and inequality rise with the displacement share. Adverse survivor wages are a much larger fiscal sensitivity.

## Context & Methods

This companion reads the committed outputs of `sensitivity_mixed_wage_adjustment.py`. The source is the FRS 2024-25 PolicyEngine dataset, simulated for 2026 at seed 0.

### Key Assumptions

The mixed grid assigns a share lambda of a 7% labour adjustment to displacement and the remainder to C-AIOE-graded wage cuts. The survivor-wage grid fixes the central displacement draw and varies only the complementarity-graded wage parameter.

In [1]:
from pathlib import Path
import pandas as pd

root = Path.cwd().parent if Path.cwd().name == 'analysis' else Path.cwd()
results = root / 'results' / 'robustness'
mixed = pd.read_csv(results / 'mixed_adjustment.csv')
wages = pd.read_csv(results / 'survivor_wage_grid.csv')

## Data

The two grids and the paper-facing metrics are shown below.

In [2]:
mixed[['lambda_displacement_share', 'displaced_weighted_m', 'exchequer_cost_bn', 'poverty_change_bhc_pp', 'gini_change_pp']].round(2)

,lambda_displacement_share,displaced_weighted_m,exchequer_cost_bn,poverty_change_bhc_pp,gini_change_pp
0,0.00,0.00,22.40,0.06,-0.37
1,0.25,0.40,20.54,0.45,-0.03
2,0.50,0.81,20.54,0.92,0.35
3,0.75,1.21,19.34,1.43,0.73
4,1.00,1.61,18.42,1.87,1.10


In [3]:
wages[['survivor_wage_effect_pct', 'exchequer_cost_bn', 'poverty_change_bhc_pp', 'gini_change_pp']].round(2)

,survivor_wage_effect_pct,exchequer_cost_bn,poverty_change_bhc_pp,gini_change_pp
0,-5.0,57.60,2.81,0.64
1,-2.6,45.32,2.34,0.78
2,0.0,31.90,2.10,0.94
3,2.6,18.42,1.87,1.10
4,5.0,5.97,1.73,1.25


## Results

Endpoint and central-cell checks ensure the new grids reproduce the existing scenarios.

In [4]:
assert mixed.loc[0, 'displaced_weighted_m'] == 0
assert abs(mixed.loc[0, 'exchequer_cost_bn'] - 22.39971328) < 1e-8
assert abs(mixed.loc[4, 'exchequer_cost_bn'] - 18.422136832) < 1e-8
central = wages.loc[wages['survivor_wage_effect_pct'].sub(2.6).abs().idxmin()]
assert abs(central['exchequer_cost_bn'] - mixed.loc[4, 'exchequer_cost_bn']) < 1e-8
'Endpoint and central-scenario reconciliation passed'

'Endpoint and central-scenario reconciliation passed'

## Takeaways

The fiscal cost remains between about £18bn and £22bn as the common labour adjustment moves from wage cuts to displacement. Poverty rises from 0.06 to 1.87 percentage points and the Gini effect changes from -0.37 to +1.10 points. At fixed central displacement, replacing the +2.6% survivor uplift with a -2.6% wage effect raises the fiscal cost from £18.4bn to £45.3bn. These are reduced-form sensitivities, not estimated equilibrium responses.